### PR TITLE
Frontoffice correction for SEO TITLE with ' or "

### PR DESCRIPTION
**Pull request by InformatiqueProg**
capture

### CODE REVIEW

File: templates/frontOffice/default/brand.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 {block name='no-return-functions' append}
 {if $brand_id}
     {loop name="brand.seo.title" type="brand" id=$brand_id limit="1"}
-    {$page_title = {$META_TITLE}}
+    {$page_title = {$META_TITLE nofilter}}
```
Comment (bibich) [inline templates/frontOffice/default/brand.html]: I think it's better to use this instead : `{$page_title = $META_TITLE}`

**Comment by InformatiqueProg** [comment]
Done !

**Comment by bibich** [comment]
Great.
Please, could you squash your commits ?

**Comment by InformatiqueProg** [comment]
squashed !

**Comment by bibich** [comment]
Thanks

**Pull request merged**